packages/editor/src/components/rich-text/index.js

```diff
@@ -839,25 +839,25 @@ export class RichText extends Component {
 		} ).body.innerHTML;
 	}
 
-	valueToFormat( { formats, text } ) {
+	valueToFormat( value ) {
```

Comment:
Why?

Where's the unit test which would have failed under the previous implementation?